### DISCUSSION TITLE
Milvus pods on pending state on EKS

### DISCUSSION THREAD

**Discussion by michaeldara** [milvus-io/milvus#21754] [Q&A and General discussion]
Deployed Milvus on EKS:
Pulsar proxy is in pending start state for ever: 
1. Error from server (BadRequest): container "milvus-2-2-2-pulsar-proxy" in pod "milvus-2-2-2-pulsar-proxy-0" is waiting to start: PodInitializing
Please help.

**Comment by LoveEachDay** [comment]
@michaeldara Could you check the status of  of `milvus-2-2-2-pulsar-bookie-init` and `milvus-2-2-2-pulsar-pulsar-init`? And also paste the complete output of `kubectl describe pods milvus-2-2-2-pulsar-proxy` .

**Comment by michaeldara** [comment]
@LoveEachDay 
**milvus-2-2-2-pulsar-bookie-init**
Error from server (BadRequest): container "milvus-2-2-2-pulsar-bookie" in pod "milvus-2-2-2-pulsar-bookie-0" is waiting to start: PodInitializing

**milvus-2-2-2-pulsar-pulsar-init**
Error from server (BadRequest): container "milvus-2-2-2-pulsar-pulsar-init" in pod "milvus-2-2-2-pulsar-pulsar-init-l5gq9" is waiting to start: PodInitializing

**milvus-2-2-2-pulsar-proxy**
Warning  FailedCreatePodSandBox  3m13s (x2917 over 10h)  kubelet  (combined from similar events): Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "f1109a18291d635aa2aadc883701064ffb47df10caf91a76629577634b0875c2": plugin type="aws-cni" name="aws-cni" failed (add): add cmd: failed to assign an IP address to container




**Here's all the Pod's status:
[mdara289@bastion-ue1a-01mq1 ~]$ kubectl get pods**
NAME                                       READY   STATUS              RESTARTS          AGE
milvus-2-2-2-datacoord-65b88fcccf-9smd5    0/1     CrashLoopBackOff    103 (100s ago)    10h
milvus-2-2-2-datanode-64d964fbf6-99k5m     0/1     CrashLoopBackOff    103 (2m10s ago)   10h
milvus-2-2-2-etcd-0                        1/1     Running             0                 10h
milvus-2-2-2-indexcoord-67b95b468b-dgqnm   0/1     Init:0/1            0                 10h
milvus-2-2-2-indexnode-857bc66f85-ch6t7    1/1     Running             0                 10h
milvus-2-2-2-proxy-66b8577f7d-nf8xl        0/1     Init:0/1            0                 10h
milvus-2-2-2-pulsar-bookie-0               0/1     Init:0/1            0                 10h
milvus-2-2-2-pulsar-bookie-1               0/1     Init:0/1            0                 10h
milvus-2-2-2-pulsar-bookie-2               0/1     Init:0/1            0                 10h
milvus-2-2-2-pulsar-bookie-init-9lxgc      0/1     Init:0/1            0                 10h
milvus-2-2-2-pulsar-broker-0               0/1     Init:0/2            0                 10h
milvus-2-2-2-pulsar-proxy-0                0/1     Init:0/2            0                 10h
milvus-2-2-2-pulsar-pulsar-init-l5gq9      0/1     Init:0/2            0                 10h
milvus-2-2-2-pulsar-recovery-0             0/1     Init:0/1            0                 10h
milvus-2-2-2-pulsar-zookeeper-0            1/1     Running             0                 10h
milvus-2-2-2-pulsar-zookeeper-1            0/1     ContainerCreating   0                 10h
milvus-2-2-2-querycoord-6489548ccc-rs8qc   0/1     Init:0/1            0                 10h
milvus-2-2-2-querynode-8cbffbfd7-s8pwb     0/1     Init:0/1            0                 10h
milvus-2-2-2-rootcoord-dbcd4bb57-h95pr     0/1     CrashLoopBackOff    103 (3m10s ago)   10h
